src/server/local/index.js

```diff
@@ -167,6 +168,13 @@ export default server => {
 
       sendDataFile(res, `${dataFilePath}.json`, next);
     })
+    .get(topicsDataPath, async ({ params }, res, next) => {
+      const { service, mediaId } = params;
```

Comment:
Regex is calling this `id` rather than `mediaId`
```suggestion
      const { service, id } = params;
```

Reply:
Do you mean where `id` gets called at `articleDataPath`? A bit confused. Line 172 is calling `mediaId` or is it because we aren't planning to use the `mediaIdRegex `in the actual implementation?

Reply:
The `params` in this function correspond with the params in the routing regular expression you have above:

`/:service(${serviceRegex})/topics/:id(${mediaIdRegex}):amp(${ampRegex})?`

Specifically, the `/topics/:id(${mediaIdRegex})` part is saying that the URL is going to have `/topics/some-id` in it, eg `/topics/c8y94j1yvr5t`

`:id(${mediaIdRegex})` is saying that this dynamic part is going to match the pattern specified in `mediaIdRegex`, and we want to use the name `id` for that variable

So, the `params` received in this function is going to have an `id` property, which will be the value the user put in that part of the URL